src/frontend/src/pages/index.js

```diff
@@ -1,190 +1,19 @@
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [telescopeUrl, numPages]);
-
-  function getNewPosts() {
-    setLoading(true);
-    setNumPages(numPages + 1);
-  }
-
-  const getPostsCount = useCallback(async () => {
-    if (!shouldCheckForNewPosts) {
-      return null;
-    }
-    try {
-      const res = await fetch(`${telescopeUrl}/posts`, { method: 'HEAD' });
-      if (!res.ok) {
-        throw new Error(res.statusText);
-      }
-      return res.headers.get('x-total-count');
-    } catch (error) {
-      console.log(error);
-    } finally {
-      setShouldCheckForNewPosts(false);
-    }
-    return null;
-  }, [telescopeUrl, shouldCheckForNewPosts]);
-
-  const callback = useCallback(async () => {
-    setCurrentNumPosts(await getPostsCount());
-  }, [getPostsCount]);
-
-  useEffect(() => {
-    savedCallback.current = callback;
-  });
-
-  // Get the initial posts count when currentNumPosts value changes (re-render)
-  useEffect(() => {
-    async function setPostsInfo() {
-      try {
-        setInitNumPosts(await getPostsCount());
-      } catch (error) {
-        console.log({ error });
-      }
-    }
-    setPostsInfo();
-  }, [getPostsCount, currentNumPosts]);
-
-  useEffect(() => {
-    function getCurrentNumPosts() {
-      setShouldCheckForNewPosts(true);
-      savedCallback.current();
-    }
-    savedCallback.current = callback;
-    // Polls every 5 minutes
-    const interval = setInterval(getCurrentNumPosts, 5 * 60 * 1000);
-    return () => clearInterval(interval);
-  }, [callback]);
-
-  useEffect(() => {
-    // Prevents alert from appearing upon page loading.
-    // Also checks whether there are new posts available
-    if (!loading && currentNumPosts !== initNumPosts && currentNumPosts !== 0) {
-      setAlert(true);
-    } else {
-      setAlert(false);
-    }
-  }, [currentNumPosts, initNumPosts, loading]);
-
-  function GenerateLoadButtonContent() {
-    if (endOfPosts) {
-      return 'No more posts.  Your turn! Add your feed...';
-    }
-    if (loading) {
-      return <Spinner />;
-    }
-    return 'Load More Posts';
-  }
-
   return (
     <PageBase title="Home">
       <Banner />
       <ScrollToTop />
-      <main className="main">
-        <Posts posts={posts} />
-
-        <Grid container spacing={0} direction="column" alignItems="center" justify="center">
-          <Grid item xs={12} className={classes.content}>
-            <Button
-              color="primary"
-              disabled={endOfPosts}
-              variant="outlined"
-              className={`${loading ? classes.activeCircle : ''}`}
-              onClick={() => getNewPosts()}
-            >
-              <GenerateLoadButtonContent />
-            </Button>
-          </Grid>
-        </Grid>
-
-        {alert ? <CustomizedSnackBar posts={currentNumPosts} message={snackbarMessage} /> : null}
-      </main>
+      <Container className="main">
```

Comment:
Do we need to wrap `<Posts />` in a `<Container />`?  What if we put `className="main"` on `<Posts className="main" />` and got rid of the wrapping `<Container>`?  Would that break anything here?

Reply:
Container often has built in padding that I see we leverage on the frontend. Removing it would require us updating internal CSS to mimic.